<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QWidget" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>533</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>838</width>
    <height>533</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>838</width>
    <height>533</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Panel Administratora</string>
  </property>
  <widget class="QLabel" name="AdminLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>5</y>
     <width>841</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Panel Administratora</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangePinAdminButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>470</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Zmień pin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AcceptAccAdminButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>470</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Zatwierdź konto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LogOutAdminButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>470</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wyloguj sie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangePasswordAdminButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Zmień hasło</string>
   </property>
  </widget>
  <widget class="QTableView" name="AdminView">
   <property name="geometry">
    <rect>
     <x>-5</x>
     <y>131</y>
     <width>841</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RefreshButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>470</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Odśwież</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
